api/src/utils/get-schema.ts

```diff
@@ -104,7 +104,13 @@ export async function getSchema(
 		const database = options?.database || getDatabase();
 		const schemaInspector = createInspector(database);
 
-		schema = await getDatabaseSchema(database, schemaInspector);
+		schema = (await getSystemCache('schema')) as SchemaOverview;
+
+		if (!schema) {
+			schema = await getDatabaseSchema(database, schemaInspector);
+			await setSystemCache('schema', schema, 86_400_000 * 7);
```

Comment:
I think this cache timeout should probably be configurable 😄 
```suggestion
			await setSystemCache('schema', schema, SCHEMA_CACHE_TIMEOUT_OR_SOMETHING_BETTER_NAMED);
```

Reply:
I agree, and yet on the other side it could even be removed and just always be cached seeing as when schema is updated the entire system cache is cleared, so alternative could be to make clear system cache easier accessible (via request call)